src/modules/launcher/Wox.Infrastructure/Image/ImageCache.cs

```diff
@@ -2,17 +2,18 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
 using System.Windows.Media;
 
 namespace Wox.Infrastructure.Image
 {
     [Serializable]
     public class ImageCache
     {
-        private const int MaxCached = 5000;
+        private const int MaxCached = 50;
         public ConcurrentDictionary<string, int> Usage = new ConcurrentDictionary<string, int>();
         private readonly ConcurrentDictionary<string, ImageSource> _data = new ConcurrentDictionary<string, ImageSource>();
-
+        private const int permissibleFactor = 2;
```

Comment:
[nit] Any reason for using `2`?

Reply:
I was going along the lines of vector resizing. The size increments/decrements are always factors of two. No other reason.